composer.json

```diff
@@ -18,6 +18,7 @@
         "phpunit/phpunit": "8.5.14",
         "php-coveralls/php-coveralls": "^2.4",
         "psalm/plugin-phpunit": "^0.15.1",
+        "psr/log": "^1.1.0",
```

Comment:
OK, now you still have to depend on `psr/log` in `require`, cause the library itself relies on the PSR logger interface.

Please add "psr/log": "^1|^2|^3" to the `require` section.